org.eclipse.buildship.ui.test/src/main/groovy/org/eclipse/buildship/ui/wizard/project/ProjectCreationWizardUiTest.java

```diff
@@ -44,7 +66,174 @@ public void defaultLocationButtonInitiallyChecked() throws Exception {
+        bot.textWithLabel(ProjectWizardMessages.Label_ProjectName).setText(TEST_PROJECT_NAME);
+
+        SWTBotButton nextButton = bot.button(IDialogConstants.NEXT_LABEL);
+        bot.waitUntil(Conditions.widgetIsEnabled(nextButton));
+        bot.button(IDialogConstants.NEXT_LABEL).click();
+
+        bot.waitUntil(Conditions.shellIsActive(NEW_GRADLE_PROJECT_WIZARD_TITLE));
+        bot.button(IDialogConstants.NEXT_LABEL).click();
+
+        // wait at the project preview page until the "< Back" button is enabled
+        bot.waitUntil(Conditions.widgetIsEnabled(bot.button(IDialogConstants.BACK_LABEL)));
+
+        File workspaceRootDir = ResourcesPlugin.getWorkspace().getRoot().getLocation().toFile();
+
+        assertTrue(workspaceRootDir.isDirectory());
+
+        File[] workspaceRootFiles = workspaceRootDir.listFiles();
+        assertTrue(workspaceRootFiles.length > 0);
+        Optional<File> firstMatch = FluentIterable.of(workspaceRootFiles)
+                .firstMatch(new FileNamePredicate(TEST_PROJECT_NAME));
+
+        // check if the "TestProject" has been created
+        assertTrue(firstMatch.isPresent());
+        File testProjectDir = firstMatch.get();
+        assertTrue(testProjectDir.isDirectory());
+        File[] testProjectFiles = testProjectDir.listFiles();
+        assertTrue(testProjectFiles.length > 0);
+
+        // also check for at least the "build.gradle" and "settings.gradle"
+        // files
+        assertTrue(FluentIterable.of(testProjectFiles).anyMatch(new FileNamePredicate("build.gradle")));
+        assertTrue(FluentIterable.of(testProjectFiles).anyMatch(new FileNamePredicate("settings.gradle")));
+
+        bot.button(IDialogConstants.BACK_LABEL).click();
+
+        // make sure that the "TestProject" is deleted in case of clicking back
+        // in the wizard
+        assertFalse(testProjectDir.exists());
+
+        // cancel the wizard
+        bot.button(IDialogConstants.CANCEL_LABEL).click();
+    }
+
+    @Test
+    public void previewPageFilesAreDeletedOnCancel() {
+        openGradleNewWizard();
+
+        bot.waitUntil(Conditions.shellIsActive(NEW_GRADLE_PROJECT_WIZARD_TITLE));
+        bot.textWithLabel(ProjectWizardMessages.Label_ProjectName).setText(TEST_PROJECT_NAME);
+
+        bot.captureScreenshot("new_wiz.png");
+
+        SWTBotButton nextButton = bot.button(IDialogConstants.NEXT_LABEL);
+        bot.waitUntil(Conditions.widgetIsEnabled(nextButton));
+        nextButton.click();
+
+        bot.waitUntil(Conditions.shellIsActive(NEW_GRADLE_PROJECT_WIZARD_TITLE));
+        bot.button(IDialogConstants.NEXT_LABEL).click();
+
+        // wait at the project preview page until the "< Back" button is enabled
+        bot.waitUntil(Conditions.widgetIsEnabled(bot.button(IDialogConstants.BACK_LABEL)));
+
+        File workspaceRootDir = ResourcesPlugin.getWorkspace().getRoot().getLocation().toFile();
+
+        System.out.println(workspaceRootDir.getAbsolutePath());
```

Comment:
This code looks very similar to `previewPageFilesAreCreatedAndDeleted`, but that method doesn't use `System.out.println`. Is this a development leftover?